ehrapy/api/preprocessing/__init__.py

```diff
@@ -1,3 +1,43 @@
-from ehrapy.api.preprocessing._data_imputation import replace_explicit
+from anndata import AnnData
+
+from ehrapy.api.preprocessing._data_imputation import (
+    _knn,
+    _mean,
+    _median,
+    _miss_forest,
+    _most_frequent,
+    _replace_explicit,
+)
 from ehrapy.api.preprocessing._scanpy_pp_api import *  # noqa: E402,F403
-from ehrapy.api.preprocessing.encoding import encode, type_overview, undo_encoding
+
+
+def impute(adata: AnnData, mode: str, copy: bool = False, **kwargs) -> AnnData:
```

Comment:
Should we really do this? Maybe we should just provide the imputation functions explicitly and get rid of this high level function. I think that it this is easier to document.